pkg/k8s/operations.go

```diff
@@ -333,6 +344,126 @@ func ApplyYaml(params ApplyYamlPrams, kubeconfig string, isLocal bool) (output s
 	return outStr, nil
 }
 
+func UpgradeInfra(manifest []byte, kubeconfig string) (string, error) {
```

Comment:
@deep-poharkar can you please add some comments before each function to describe what the function does, also can you please add some more comments in function body for better understanding of code base

Reply:
done PTAL